pkg/config/mesh/mesh_test.go

```diff
@@ -91,6 +89,11 @@ defaultConfig:
 	}
 	// Verify overrides
 	got, err = mesh.ApplyMeshConfigDefaults(`
+serviceSettings: 
```

Comment:
This isn't really tested right? You just verify this is not crashing I guess, not actually check its applied correctly?

Reply:
yup exactly ... I really only added this so I could get the yaml right for the mc test.